workloads/kube-burner/run_nodedensity-cni-networkpolicy_test_fromgit.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/bash
+
+WORKLOAD_TEMPLATE=workloads/node-density-cni-networkpolicy/node-density-cni-networkpolicy.yml
+METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+NODE_COUNT=${NODE_COUNT:-4}
+PODS_PER_NODE=${PODS_PER_NODE:-250}
+export WORKLOAD=node-density-cni-networkpolicy
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+label_nodes heavy
```

Comment:
why are we labeling nodes w/ "heavy"? same for the other test.

Reply:
ctrl C+v obviously, good catch

Reply:
well we'd still need a label - perhaps a different label but we assigns pods to nodes based on the label right? This is still a kubelet-density style of test.

Reply:
Just changed the label to cni.